zql/docs/search-syntax/README.md

```diff
@@ -357,9 +357,29 @@ dns   1521912892.637238 CN9X7Y36SH6faoh8t 10.47.8.10 58340     10.0.0.100 53
 
 ### Wildcard Field Names
 
-Since the data type of the value is considered in field/value matches, it's possible to search for the value across any fields of the value's type by entering a wildcard (`*`) in place of the field name.
+Since the data type of the value is considered in field/value matches, it's possible to search for the value across any fields of the value's type by entering a wildcard in place of the field name. Two wildcard operators are available depending on how broad you want your search to be. The `*` operator matches all top-level fields of the value's type, and the `**` operator additionally matches such values when they appear nested within records.
```

Comment:
```suggestion
It's possible to search for the typed value across _any_ field by entering a wildcard in place of the field name. Two wildcard operators are available depending on how broad you want your search to be. The `*` operator matches all top-level fields of the value's type, and the `**` operator additionally matches such values when they appear nested within records.
```

Reply:
s/across any field/across all fields/

Reply:
@henridf @nwt: I've incorporated your feedback, but made a small additional adjustment. I respect that the leading "Since the data type..." text was arguably redundant, so I understand the instinct to remove it. At the same time (and maybe I'm not giving our readers enough credit here) I'm concerned that the shorthand "typed value" could be misinterpreted as "the value I typed into the Search box in Brim" as opposed to "the value of the data type that the ZQL parser interpreted the value to be". I don't expect confusion from users who do programming, but I also expect some non-programmer users coming to Brim/ZQL after using incumbent search tools may need more of a helping hand. Anyway, please do give it another look and let me know what you think.